contracts/contracts/strategies/NativeStaking/CompoundingValidatorManager.sol

```diff
@@ -772,8 +772,9 @@ abstract contract CompoundingValidatorManager is Governable, Pausable {
                 params.validatorContainerProof
             );
 
-            // for each validator
-            for (uint256 i = 0; i < verifiedValidatorsCount; ++i) {
+            // for each validator in reserve order so we can pop off exited validators at the end
+            for (uint256 i = verifiedValidatorsCount; i > 0; ) {
```

Comment:
you could also put the whole for statement in the same line: 
```
for (uint256 i = verifiedValidatorsCount - 1; i > 0; i--) {
```
The `verifiedValidatorsCount` will always be >= 1

Reply:
Ah, yes, because of the earlier `if (verifiedValidatorsCount > 0) {`

Thanks

Reply:
Maybe not. The last loop when i = 0 is skipped.

Reply:
yeah true should be: 
```
for (uint256 i = verifiedValidatorsCount - 1; i >= 0; i--) {
```